src/components/v5/shared/SearchSelect/types.ts

```diff
@@ -22,10 +23,10 @@ export interface SearchSelectOptionProps {
 }
 
 export interface SearchSelectOption {
-  label: MessageDescriptor | string;
-  value: string;
+  label?: MessageDescriptor | string | null;
+  value?: string | null;
```

Comment:
Value should be required

Reply:
yes you're right but what this hook `useGetVerifiedMembersQuery` returns can be null | string | undefined

Reply:
https://github.com/JoinColony/colonyCDapp/pull/1064#discussion_r1354228669